widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -33,6 +33,10 @@
  * This class is a starting point for integration with Glia Widgets SDK
  */
 public class GliaWidgets {
+
+    /**
+     * TODO
+     */
```

Comment:
This variable should be leftover, AFAIK this variable is not in use.

Reply:
I think you are right. I have not found either the constant usage or the raw string usage in the project. 
I will delete it.